Src/RE2.Managed/NativeMethods.cs

```diff
@@ -76,6 +76,30 @@ public static unsafe int Matches(int regexIndex, String8Interop text, int fromTe
                 : NativeMethodsX86.Matches(regexIndex, text, fromTextIndex, matches, matchesLength, timeoutMilliseconds);
         }
 
+        public static unsafe void GetRegexSetup(StringUtf8 pattern, ulong* numCapturingGroups, ulong* numNamedCapturingGroups, ulong* numSubmatches, ulong* groupNameHeadersBufferSize, ulong* groupNamesBufferSize, ulong* submatchesBufferSize)
+        {
+            if (Environment.Is64BitProcess)
+            {
+                NativeMethodsX64.GetRegexSetup(pattern, numCapturingGroups, numNamedCapturingGroups, numSubmatches, groupNameHeadersBufferSize, groupNamesBufferSize, submatchesBufferSize);
+            }
+            else
+            {
+                throw new NotImplementedException("x86 not implemented");
```

Comment:
>NotImplementedException [](start = 26, length = 23)

since the implementation is the same, u just need to a dd the same files to the vcxproj.

Reply:
The implementation will probably be very similar or identical. It may not be identical due to performance differences between x64 and x86 with different sized numeric types. I'll need to do the analysis and refactor this.

Reply:
Moved to backlog: [https://github.com/microsoft/sarif-pattern-matcher/issues/386](https://github.com/microsoft/sarif-pattern-matcher/issues/386)